spec/index.html

```diff
@@ -504,6 +504,7 @@ <h2>Simple Interpretations</h2>
     the referent of the subject and object of any true triple will be in IR;
     so any IRI which occurs in a graph both as a predicate and as a subject or object
     will denote something in the intersection of IP and IR.</p>
+  <p>We observe that no IRI, not even those in the <code>rdf:</code> namespace, has any special semantic condition associated with it in a simple interpretation.</p>
```

Comment:
Shorter lines are better.
```suggestion
  <p>We observe that no IRI, not even those in the <code>rdf:</code> namespace,
  has any special semantic condition associated with it in a simple interpretation.</p>
```